packages/salesforcedx-utils-vscode/test/unit/context/workspaceContextUtil.test.ts

```diff
@@ -8,49 +8,9 @@
 import { AuthInfo, Connection } from '@salesforce/core';
 import { expect } from 'chai';
 import { join } from 'path';
-import * as proxyquire from 'proxyquire';
```

Comment:
proxyquire is no longer required due to us being able to virtually mock vscode for unit tests.